module-2/backend/week_two.md

```diff
@@ -5,19 +5,46 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+---It's a wrapper for SQL that allows you to easily access a database within an application with easy commands.
+
 2. What is a migration?
+--It's a set of instructions for creating a database.
+
 3. How does a table relate to a model?
+--We have a model for each table.  A model is a class that does calculations and operations on the data.
+
 4. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+--They are class methods, which is why they're in the models which are classes.  In Blogger, we had an Article class and an Attachment class.  Each attachment "belonged_to" an article and an article could "have_many" attachments.
```

Comment:
Tricky question: these are actually instance methods. The important thing to remember is that you need a specific instance before you can use these. For example:

```ruby
kitten_in_tree = Article.find(1) # assigns the Article with an `id` of 1 to the variable `kitten_in_tree`.
attached_image = kitten_in_tree.attachment # uses the `attachment` method that `has_many :attachments` gives us in our Article class.
```